### PR TITLE
fix/1939 swap presentation/none roles

### LINKED ISSUE #1939: Suggestions: swap most content from #presentation and #none role section
# swap most content from #presentation and #none role sections

All of the user reqs and explanation are in the original "presentation" role section of the spec, but we've started emphasizing a preference for the "none" role. To keep in line with that, the content in the "none" and "presentation" sections of the spec should be swapped, along with any required editorial caveats and massaging.

# Links
https://w3c.github.io/aria/#none
https://w3c.github.io/aria/#presentation

# Does the issue exist in [the editors draft](https://w3c.github.io/aria/) 
Yes

### PR DESCRIPTION
**Pull request by ariellalgilmore**
Closes #1939 

worked with @andreancardona 🎉 

- Swapped `presentation` role information with `none` role information to emphasize the users to use `none` role. 
- Adjusted text when `presentation` role is mentioned to include `none/presentation role`. 
- Always put `none` first and updated examples to use `none` over `presentation`

***
<a href="https://pr-preview.s3.amazonaws.com/ariellalgilmore/aria/pull/1945.html" title="Last updated on May 25, 2023, 9:50 PM UTC (fb01b55)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/aria/1945/fdfbf10...ariellalgilmore:fb01b55.html" title="Last updated on May 25, 2023, 9:50 PM UTC (fb01b55)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6380,141 +6511,10 @@ <h4>Plain HTML or Polyfill DOM Result of the MathML Quadratic Formula</h4>
...
 				<div class="note" id="role-presentation-note-none">
-					Note regarding the ARIA 1.1 <rref>none</rref> role.
-					<p>In ARIA 1.1, the working group introduced <rref>none</rref> as a synonym to the <code>presentation</code> role, due to author confusion surrounding the intended meaning of the word "presentation" or "presentational." Many individuals erroneously consider <code>role="presentation"</code> to be synonymous with <code>aria-hidden="true"</code>, and we believe <code>role="none"</code> conveys the actual meaning more unambiguously.</p>
+					Note regarding the ARIA 1.1 <code>none</code> role.
+					<p>In ARIA 1.1, the working group introduced <code>none</code> as a synonym to the <rref>presentation</rref> role, due to author confusion surrounding the intended meaning of the word "presentation" or "presentational." Many individuals erroneously consider <code>role="presentation"</code> to be synonymous with <code>aria-hidden="true"</code>, and we believe <code>role="none"</code> conveys the actual meaning more unambiguously.</p>
```
Comment (cookiecrook) [inline index.html:6515]:
```suggestion
					Note regarding the ARIA 1.1 <rref>none</rref> role.
					<p>In ARIA 1.1, the working group introduced <rref>none</rref> as the preferred synonym to the <rref>presentation</rref> role, due to author confusion surrounding the intended meaning of the word "presentation" or "presentational." Many individuals erroneously consider <code>role="presentation"</code> to be synonymous with <code>aria-hidden="true"</code>, and we believe <code>role="none"</code> conveys the actual meaning more unambiguously.</p>
```

File: index.html [hunk 2]
```diff
@@ -13690,16 +13690,16 @@ <h3>Presentational Roles Conflict Resolution</h3>
 		<p>There are a number of ways presentational role conflicts are resolved.</p>
 		<p>User agents MUST NOT expose [=elements=] having explicit or inherited presentational role in the accessibility tree, with these exceptions:</p>
 		<ul>
-			<li>If an element is focusable, user agents MUST ignore the <code>presentation</code>/<code>none</code> role and expose the element with its implicit role, in order to ensure that the element is <a>operable</a>.</li>
+			<li>If an element is focusable, user agents MUST ignore the <code>none</code>/<code>presentation</code> role and expose the element with its implicit role, in order to ensure that the element is <a>operable</a>.</li>
```
Comment (cookiecrook) [inline index.html:13693]:
```suggestion
			<li>If an element is focusable, user agents MUST ignore the <rref>none</rref>/<rref>presentation</rref> role and expose the element with its implicit role, in order to ensure that the element is <a>operable</a>.</li>
```

File: index.html [hunk 3]
```diff
@@ -13690,16 +13690,16 @@ <h3>Presentational Roles Conflict Resolution</h3>
...
 			<li>If an <a href="#mustContain">allowed child element</a> has an explicit non-presentational role, user agents MUST ignore an inherited presentational role and expose the element with its explicit role. If the action of exposing the explicit role causes the accessibility tree to be malformed, the expected results are undefined.</li>
-			<li>If an element has global WAI-ARIA states or properties, user agents MUST ignore the <code>presentation</code> role and instead expose the element's implicit role. However, if an element has only non-global, role-specific WAI-ARIA states or properties, the element MUST NOT be exposed unless the presentational role is inherited and an explicit non-presentational role is applied.</li>
+			<li>If an element has global WAI-ARIA states or properties, user agents MUST ignore the <code>none</code>/<code>presentation</code> role and instead expose the element's implicit role. However, if an element has only non-global, role-specific WAI-ARIA states or properties, the element MUST NOT be exposed unless the presentational role is inherited and an explicit non-presentational role is applied.</li>
```
Comment (cookiecrook) [inline index.html:13695]:
```suggestion
			<li>If an element has global WAI-ARIA states or properties, user agents MUST ignore the <rref>none</rref>/<rref>presentation</rref> role and instead expose the element's implicit role. However, if an element has only non-global, role-specific WAI-ARIA states or properties, the element MUST NOT be exposed unless the presentational role is inherited and an explicit non-presentational role is applied.</li>
```

**Review by cookiecrook — APPROVED** [review]
Minor editorial suggestions ready to merge, and one item I propose be addressed as a separate issue, so that this PR is not unnecessarily delayed.

Otherwise, LGTM.

File: index.html [hunk 4]
```diff
@@ -5964,10 +5964,141 @@ <h4>Plain HTML or Polyfill DOM Result of the MathML Quadratic Formula</h4>
...
+				<p class="note">Only the implicit native semantics of elements that correspond to <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> <a href="#mustContain">Allowed Accessibility Child Roles</a> are removed. All other content remains intact, including nested tables or lists, unless those elements also have an explicit role of <code>none</code>/<code>presentation</code> specified.</p>
+				<p>For example, according to an accessibility <abbr title="Application Programing Interface">API</abbr>, the following markup elements would appear to have identical role semantics (no roles) and identical content.</p>
```
[the remark this answers is not in the document]
Comment (ariellalgilmore) [reply index.html:6006]: thanks for the review @cookiecrook! Will open another editorial issue to look at this text

[the remark this answers is not in the document]
Comment (ariellalgilmore) [reply index.html:6006]: https://github.com/w3c/aria/issues/1946

**Review by adampage — APPROVED** [review]
Looks terrific. 🚀

**Comment by spectranaut** [comment]
@pkra - seems like this might be ready to merge?

**Pull request merged**